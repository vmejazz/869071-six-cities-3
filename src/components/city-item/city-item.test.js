import React from "react";
import renderer from "react-test-renderer";
import CityItem from "./city-item";

it(`<CityList /> should render cityes buttons`, () => {
  const tree = renderer
      .create(
          <CityItem
            city={`Moscow`}
            activeCity={`Paris`}
            onCityClick={() => {}}
          />
      )
      .toJSON();

  expect(tree).toMatchSnapshot();
});
