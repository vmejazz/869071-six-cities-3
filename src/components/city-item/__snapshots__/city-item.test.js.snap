// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CityList /> should render cityes buttons 1`] = `
<li
  className="locations__item"
  onClick={[Function]}
>
  <a
    className="locations__item-link tabs__item "
    href="#"
  >
    <span>
      Moscow
    </span>
  </a>
</li>
`;
